test/jll.jl

```diff
@@ -12,12 +12,13 @@ module TestJLL end
     @test project["deps"] == Dict("JLLWrappers" => "692b3bcd-3c85-4b1f-b108-f13ce0eb3210",
                                   "Artifacts"   => "56f22d72-fd6d-98f1-02f0-08ddc0907c33",
                                   "Pkg"         => "44cfe95a-1eb2-52ea-b672-e2afdf69b78f",
+                                  "Preferences" => "21216c6a-2e73-6563-6e65-726566657250",
```

Comment:
Like above, after https://github.com/JuliaPackaging/JLLWrappers.jl/pull/29 this isn't necessary
```suggestion
```